### DISCUSSION TITLE
Version 0.7.6

### DISCUSSION THREAD

**Discussion by aaronburtle** [Azure/data-api-builder#1523] [Announcements]
Announcing release [0.7.6 for Data Api builder](https://github.com/Azure/data-api-builder/releases/tag/v0.7.6) for Azure Databases.

The following issues were newly fixed as part of this release:

- Address filter access deny issue for Cosmos by @tarazou9 in https://github.com/Azure/data-api-builder/pull/1436
- Database policy support for PUT/PATCH operations - MsSql by @ayush3797 in https://github.com/Azure/data-api-builder/pull/1428
- Adding Merge config capabilities by @abhishekkumams in https://github.com/Azure/data-api-builder/pull/1446
- Executing GraphQL and REST Mutations in a transaction by @severussundar in https://github.com/Azure/data-api-builder/pull/1451
- Add support for Cosmos field level auth by @tarazou9 in https://github.com/Azure/data-api-builder/pull/1468
- Initial Support for OpenAPI v3.0.1 description document creation by @seantleonard in https://github.com/Azure/data-api-builder/pull/1462
- Adding useragent to cosmos client by @mbhaskar in https://github.com/Azure/data-api-builder/pull/1486
- Hot fix merge config by @abhishekkumams in https://github.com/Azure/data-api-builder/pull/1507
- Bug fix for CosmosDB field auth when graphql is true, include is * by @tarazou9 in https://github.com/Azure/data-api-builder/pull/1516

**Comment by dgcaron** [comment]
i am running into an issue with this version --> #1538 
there also seems to be a regression issue with the filter error fixed in #1436

**Reply by Aniruddh25** [reply]
As the investigation from #1538 indicates, this is by design as per the filter field authorization support added for CosmosDb_NoSQL. 
@dgcaron, Please update your configuration to avoid running into issues with the latest version. Thanks!
